src/screens/InboundOrder/index.tsx

```diff
@@ -22,40 +29,42 @@ const InboundOrder = () => {
         showPopup({
           title: data.errorMessage ? 'Inbound order details' : null,
           message:
-              data.errorMessage ??
-              `Failed to load inbound order details value ${id}`,
+            data.errorMessage ??
+            `Failed to load inbound order details value ${id}`,
           positiveButton: {
             text: 'Retry',
             callback: () => {
               dispatch(fetchInboundOrderList(callback, id));
-            },
+            }
           },
-          negativeButtonText: 'Cancel',
+          negativeButtonText: 'Cancel'
         });
       } else {
         if (data && Object.keys(data).length !== 0) {
-
           state.inboundOrder = data.filter((item: any) => {
             return (
-                item.status === 'SHIPPED' || item.status === 'PARTIALLY_RECEIVED'
+              (item?.destination?.name?.toLowerCase() ===
```

Comment:
Why the usage of the `toLowerCase()`? I think it should work without it fine.

Reply:
@awalkowiak it's a string and in case it's case sensitive that's why I have usage lowercase.

Reply:
@hellochirag there should be no issue with it. And you can use id here instead of a name, that will be safe.